recipes/gnuradio-iio/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "gnuradio-iio" %}
+{% set hash = "733c8a05e74b7d10fbaef502cc82d025ae35a1fb" %}
+{% set version = "0.3.20200508+gr38" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #url: https://github.com/analogdevicesinc/gr-iio/archive/v{{ version }}.tar.gz
+  #sha256:
+  git_url: https://github.com/analogdevicesinc/gr-iio.git
+  git_rev: {{ hash }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('gnuradio-iio', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja  # [win]
+    - pkg-config  # [not win]
+  host:
+    - bison  # [not win]
+    - flex  # [not win]
+    - gnuradio-core
+    - libad9361-iio
+    - libiio
+    - python
+    - swig
+    - volk
+    - winflexbison  # [win]
+  run:
+    - python
+    # pulled in by gnuradio-core and linked against, so include explicitly
+    - boost-cpp
+    - log4cpp
```

Comment:
boost-cpp and log4cpp should be in the host deps as well, since they are compiled against